packages/settings/src/ui/settings-ui-network-list.tsx

```diff
@@ -43,7 +44,7 @@ export function SettingsUiNetworkList({
               description="This action cannot be reversed."
               title="Are you sure you want to delete this network?"
             >
-              <Button size="icon" title={t(($) => $.actionDelete)} variant="outline">
+              <Button disabled={isLastNetwork} size="icon" title={t(($) => $.actionDelete)} variant="outline">
```

Comment:
I think the best way to go about this is to disable the active network. Combined with the logic in the db layer this seems like the easiest way to go about it.